modules/standard/Random.chpl

```diff
@@ -293,8 +337,21 @@ module Random {
         compilerError("NPBRandomStream(eltType=", eltType:string, 
                       ") can only be used to fill arrays of ", eltType:string);
       }
-        
-      pragma "no doc"
+
+      /*
+         Returns an iterable expression for generating
+         D.numIndices random numbers. The RNG state will
+         be immediately advanced by D.numIndices before
+         this function yields any values.
+
+         The iterable expression can be usefully used
+         in a parallel context, whether standalone or
+         zippered.
+
+         :arg D: a domain
+         :arg resultType: the type of number to yield
+         :return: an iterable expression yielding random resultType values
+       */
```

Comment:
I'm still not in particular favor of exposing iterate().  It's such a weird routine, requiring a domain...  has a weird name...  relies on returning an iterator...  Seems like the type of thing it'd be nicer to get right and then expose to the user rather than exposing a half-baked version to the user and then being forced to support it for all time or go to the trouble of deprecating it in the future...